docs/repo/property-pages/README.md

```diff
@@ -1,6 +1,6 @@
 ﻿# Project Property Pages
 
-This documentation details the updated Project Properties UI and associated back end, added to Visual Studio in 2021.
```

Comment:
I think this comment is correct. This is not talking about the version for Visual Studio but the timeframe of when it was added

Reply:
It's confusing as it's read -- we never talk about timeframes we talk about when/what products they are introduced.

Reply:
The way the updated version reads makes sense